packages/client/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,20 @@
+/* istanbul ignore file */
+import React from "react";
+
+import Layout from "../components/Layouts/BaseLayout";
+import { isProduction } from "../config";
+import { DebugCheckersTable } from "../debug";
+
+const HomePage: React.FC = () => {
+  if (isProduction) {
+    localStorage.setItem("doNotTrack", "true");
```

Comment:
Why put `doNotTrack` on?

Reply:
It has been there for a long time. It was part of the DevHomePage.
People that are on Home should not be tracked.
Or am I missing something?

Reply:
![image](https://user-images.githubusercontent.com/7781865/98707431-5263f280-2380-11eb-9d99-a16cf1f94035.png)

This is code is now also executed in the root on `/`, while it should stop tracking when entering `/test`